src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -188,30 +155,28 @@ export const gravityStitchingEnvironment = (
       }
 
       extend type ViewingRoomPublishedNotificationItem {
-        viewingRoomsConnection(first: Int, after: String, last: Int, before: String): ViewingRoomsConnection
+        viewingRoomsConnection(
+          first: Int
+          after: String
+          last: Int
+          before: String
+        ): ViewingRoomsConnection
       }
 
       extend type Viewer {
-        viewingRoomsConnection(first: Int, after: String, statuses: [ViewingRoomStatusEnum!], partnerID: ID): ViewingRoomsConnection
+        viewingRoomsConnection(
+          first: Int
+          after: String
+          statuses: [ViewingRoomStatusEnum!]
+          partnerID: ID
+        ): ViewingRoomsConnection
       }
 
       # Mutation Payloads
       extend type CreateUserAddressPayload {
         me: Me
       }
 
-      extend type CreateSavedSearchPayload {
-        me: Me
-      }
-
-      extend type UpdateUserAddressPayload {
```

Comment:
sup with these

Reply:
Huh that looks like a mistake! Fixed and force pushed.